neo/rawio/maxwellrawio.py

```diff
@@ -0,0 +1,221 @@
+                ch_name = f'ch{chan_id} elec{elec_id}'
+                offset_uV = 0
+                sig_channels.append((ch_name, str(chan_id), sr, 'uint16', 'uV',
+                                     gain_uV, offset_uV, stream_id))
+
+            self._signals[stream_id] = sigs
+            max_sig_length = max(max_sig_length, sigs.shape[1])
+
+        self._t_stop = max_sig_length / sr
+
+        sig_channels = np.array(sig_channels, dtype=_signal_channel_dtype)
+
+        spike_channels = []
+        spike_channels = np.array(spike_channels, dtype=_spike_channel_dtype)
+
+        event_channels = []
+        event_channels = np.array(event_channels, dtype=_event_channel_dtype)
+
+        self.header = {}
+        self.header['nb_block'] = 1
+        self.header['nb_segment'] = [1]
+        self.header['signal_streams'] = signal_streams
+        self.header['signal_channels'] = sig_channels
+        self.header['spike_channels'] = spike_channels
+        self.header['event_channels'] = event_channels
+
+        self._generate_minimal_annotations()
+        bl_ann = self.raw_annotations['blocks'][0]
+        bl_ann['maxwell_version'] = version
+
+    def _segment_t_start(self, block_index, seg_index):
+        return 0.
+
+    def _segment_t_stop(self, block_index, seg_index):
+        return self._t_stop
+
+    def _get_signal_size(self, block_index, seg_index, stream_index):
+        stream_id = self.header['signal_streams'][stream_index]['id']
+        sigs = self._signals[stream_id]
+        return sigs.shape[1]
+
+    def _get_signal_t_start(self, block_index, seg_index, stream_index):
+        return 0.
+
+    def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop,
+                                stream_index, channel_indexes):
+        stream_id = self.header['signal_streams'][stream_index]['id']
+        sigs = self._signals[stream_id]
+
+        if i_start is None:
+            i_start = 0
+        if i_stop is None:
+            i_stop = sigs.shape[1]
+
+        if channel_indexes is None:
+            channel_indexes = slice(None)
+
+        try:
+            if self._old_format:
+                sigs = sigs[self._channel_slice, i_start:i_stop]
+                sigs = sigs[channel_indexes]
+            else:
+                sigs = sigs[channel_indexes, i_start:i_stop]
+        except OSError as e:
+            print('*' * 10)
+            print(_hdf_maxwell_error)
+            print('*' * 10)
+            raise(e)
+        sigs = sigs.T
+
+        return sigs
+
+
+_hdf_maxwell_error = """Maxwell file format is based on HDF5.
+The internal compression need a custum plugin!!!
+This is a big pain for the end user.
+You, as a end user, should ask Maxwell company to change this.
+Please visit this page and install what is missing:
+https://share.mxwbio.com/d/4742248b2e674a85be97/
+Then, you will need to do something like this in your code
+os.environ['HDF5_PLUGIN_PATH'] = '/path/to/cutum/hdf5/plugin/'
```

Comment:
are you sure the user is supposed to overwrite this variable if it already exists? Or should this rather be a list and the new entry needs to be appended?

Reply:
I  most cases this ENV do not exists.
So 99% users, this will help.
For hdf5-advanced users, they will anderstand what to do.

Reply:
ok, let's hope they raise an issue when they encounter problems related to this...